Fullstack-Development/Communities/Readme.md

```diff
@@ -4,4 +4,5 @@
 - [Eddiehub](http://discord.eddiehub.org/)
 - [Geek Around Community](https://discord.io/geekaroundcommunity)
 - [Hack Club Hackerabad](https://discord.hackerabad.tech/)
-- [MLH (Hackathons)](https://discord.gg/mlh)
+- [MLH (Hackathons)](https://discord.gg/mlh/)
+- [Hacker News](https://news.ycombinator.com)
```

Comment:
Hackernews is not a community, it is a Social Media website that is focused on tech and entrepreneurship.